app/src/pages/trip_reports/2022_5040.md

```diff
@@ -3,9 +3,27 @@
 [photos here](https://photos.app.goo.gl/TBafftjf7kaebusH9)
 
 ## Trip Plan
+
 ## Day 1 (approach)
+- left Victoria at 5am
+- started hiking from car at 9am (exactly on time)
+- at trailhead at 12:30
+- at hut at 5pm (sunset)
```

Comment:
snowshoers arrived at 3:45 PM (I have a pic of the arrival with that timestamp)

Reply:
show off :p

Reply:
lol